sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs

```diff
@@ -5170,6 +5170,28 @@ public void Cfg_PropertyPatternClause_Nested()
 
         #endregion
 
+        #region Instance creation
+
+        [TestMethod]
+        [TestCategory("CFG")]
+        public void Cfg_New()
+        {
+            var cfg = Build(@"var x = new Object()");
+
+            VerifyMinimalCfg(cfg);
+            VerifyAllInstructions(cfg.EntryBlock, "new Object()", "x = new Object()");
+        }
+
+        [TestMethod]
+        [TestCategory("CFG")]
+        public void Cfg_New_TargetTyped()
```

Comment:
why did you add a test with only the "target typed new" syntax, and not will the rest of C# 9 syntaxes?

Reply:
Mainly because the scope of this PR is CBDE. I just documented the root cause of this known failure to isolate it. I wasn't able to break CFG with other syntaxes.